website_sale_attribute_filter_category/__manifest__.py

```diff
@@ -12,13 +12,11 @@
     'application': False,
     'installable': True,
     'depends': [
+        'website_sale_product_attribute_filter_visibility',
```

Comment:
Why this extra dependency?

Reply:
Oh my goodness!!! What dependency?? :smile:

Reply:
So that means?

Reply:
OK, I see you have pushed something.